### ISSUE TITLE
Loading module from _nuxt/empty.js was blocked when enabling Vite

### ISSUE THREAD

**Issue by HendrikJan** [nuxt/bridge#569]
### Environment

------------------------------
- Operating System: `Linux`
- Node Version:     `v14.19.3`
- Nuxt Version:     `2.16.0-27720022.54e852f`
- Nitro Version:    `0.5.4`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `bridge`, `ssr`, `serverHandlers`, `devServerHandlers`, `buildModules`
- Runtime Modules:  `-`
- Build Modules:    `()`, `@nuxt/bridge@3.0.0-27740081.058a6c1`
------------------------------

### Reproduction

1. Open the default CodeSandbox template: https://codesandbox.io/p/github/nuxt/starter/tree/v2-bridge-codesandbox
2. Replace `nuxt.config.js` with this:

```javascript
import { defineNuxtConfig } from "@nuxt/bridge";

export default defineNuxtConfig({
  bridge: {
    vite: true
  },

  ssr: false
});
```
(or open this CodeSandbox where I already replaced the config: https://codesandbox.io/s/nuxt-bridge-vite-empty-error-ee3gjg?file=/nuxt.config.js)

3. In Firefox, open the console (the error does not show in the sandbox console) and see the error:
```
Loading module from “https://ee3gjg.sse.codesandbox.io/_nuxt/empty.js” was blocked because of a disallowed MIME type (“text/html”).
Loading failed for the module with source “https://ee3gjg.sse.codesandbox.io/_nuxt/empty.js”.
```

4. In Chrome open the console and see the error:
```
empty.js: Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

5. See that the page does not load

### Describe the bug

When using `vite: true` and `ssr: false` in Nuxt/bridge, the page does not load and in your browser console you find an error telling that the file `empty.js` was blocked because it has the wrong MIME-type.

**Comment by saideepesh000** [comment]
👍 , facing the same issue

**Comment by smortexa** [comment]
We are facing the same issue too.

**Comment by mahsarajabpour** [comment]
Facing the same issue.

**Comment by zhancheng** [comment]
Facing the same issue.

**Comment by yulafezmesi** [comment]
same here

**Comment by ad-on-is** [comment]
Same here, setting `ssr:false` gives this error.

**Comment by joel-wenzel** [comment]
@danielroe Any ideas here? Happy to look into it otherwise

**Comment by emretepedev** [comment]
Same issue here.

**Comment by danielroe** [comment]
Would welcome it, thank you @joel-wenzel 🙏

**Comment by JeffCharron** [comment]
I just ran into this same issue

**Comment by schirrel** [comment]
Why does nuxt bridge have this problem with ssr false? it is not suposed to work with ssg?

**Comment by JerryKusenberger** [comment]
I am also experiencing this. The workaround from @AndreyYolkin did not work for me.

**Comment by AndreyYolkin** [comment]
> I am also experiencing this. The workaround from @AndreyYolkin did not work for me.

Do you have nitro enabled? And also: webpack or vite? Will try to help

**Comment by JerryKusenberger** [comment]
Yes I have nitro enabled. In my bridge options I set vite: true. Everything works if I set vite: false

**Comment by JerryKusenberger** [comment]
I just created an empty repo with just nuxt and nuxt bridge installed.

This is all that is in the nuxt config. If I set SSR to true the problem goes away.

`import { defineNuxtConfig } from '@nuxt/bridge'

export default defineNuxtConfig({
  bridge: {
    nitro: true,
    vite: true, 
  },
 ssr: false,
})
`

**Comment by AndreyYolkin** [comment]
https://github.com/AndreyYolkin/nuxt-bridge-app-spa this seems to be ok (despite the ENOENT error, which I've already submitted as https://github.com/nuxt/bridge/issues/734)

**Comment by JerryKusenberger** [comment]
thanks @AndreyYolkin I'm one step closer. Now experiencing the same ENOENT issue.

**Comment by wattanx** [comment]
The issue has been resolved in https://github.com/nuxt/bridge/pull/775.
cc: @danielroe
